app/scripts/app.config.js

```diff
@@ -30,15 +30,15 @@ angular.module('confRegistrationWebApp')
       },
       vars: {
         development: {
-          apiUrl: 'https://api.stage.eventregistrationtool.com/eventhub-api/rest/',
+          apiUrl: 'http://localhost:8080/eventhub-api/rest/',
```

Comment:
Did you mean to commit this?

Reply:
My mistake. I keep inadvertently including that in the commit. Resolved.